fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryTest.java

```diff
@@ -69,7 +69,7 @@ public void testGetIncrement() {
         assertEquals(1, directory.getIncrement());
         assertEquals(2, directory.getIncrement());
         assertCorrectMetricCount(LuceneEvents.Counts.LUCENE_GET_INCREMENT_CALLS, 2);
-        directory.getContext().commit();
+        directory.getCallerContext().commit();
```

Comment:
In the (potential) followup where `getCallerContext` is removed, you could change the `context` created in `setUp`, and use that in all of these usages.
Although, looking at this test, and some others, it would probably make sense to remove the fields for `directory` and just create the `context` & directory in a `try-with-resources` in the test (as is done in this test), and then reference the contexts that we are creating.